src/index.ts

```diff
@@ -2,7 +2,7 @@ export * as networkIDs from './constants/networkIDs';
 export * from './serializable';
 export { Utxo } from './serializable/avax/utxo';
 export * from './signer';
-export * from './utils';
```

Comment:
Do we care that this is backwards breaking? Same for moving crypto out of utils. I am fine with it, but curious what our approach to compatibility is now that this is public.

Reply:
I think a minor release is the appropriate method here. I intentionally made it breaking since we're using more than one type of private key